Projects/UOContent/Items/Weapons/Abilities/DoubleShot.cs

```diff
@@ -7,9 +9,11 @@ public class DoubleShot : WeaponAbility
     {
         public override int BaseMana => 30;
 
+        public override bool RequiresTactics(Mobile from) => false;
+
         public override bool CheckSkills(Mobile from)
         {
-            if (GetSkill(from, SkillName.Ninjitsu) < 50.0 && GetSkill(from, SkillName.Bushido) < 50.0)
+            if(Math.Max(GetSkill(from, SkillName.Ninjitsu), GetSkill(from, SkillName.Bushido)) < 50)
```

Comment:
I think this change is less readable and doesn't have a performance gain. Did you have a specific reason for this change? 🤔